lotr_homework/js/app.js

```diff
@@ -47,20 +48,42 @@ const makeMiddleEarth = () => {
   console.log("Trying to make middle earth.");
 
   // 1. create a section tag with an id of middle-earth
-
+  const $body = $('body');
+  const $section = $('<section id="middle-earth"></section>');
+  const $article = $('<article></article>')
+  $body.append($section);
+  
+  
   // 2. append the section to the body of the DOM.
 
+  //check above
+
   // 3. use a for loop to iterate over the lands array that does the following:
+    
+
+    for (let i = 0; i < lands.length; i++){  
+      
+      $article.attr('id', lands[i]);
+      $article.append('<h1>' + lands[i] +'</h1>');
+      $section.append($article);
+  }
+};
+  //sorry, I dislike the comments between the loop. Makes it harder for me to read. 
 
   //   3a. creates an article tag (there should be one for each land when the loop is done)
 
+  //check
   //   3b. gives each land article an `id` tag of the corresponding land name
 
+  //check
   //   3c. includes an h1 with the name of the land inside each land article
 
+  //check
   //   3d. appends each land to the middle-earth section
```

Comment:
your code appends all 3 lands to one article tag instead of creating an article tag for each land.  how were you able to do any of the other questions?